### PR TITLE
Extra covariates

### PR DESCRIPTION
**Pull request by SStock1**

### CODE REVIEW

File: causalpy/pymc_experiments.py [hunk 1]
```diff
@@ -59,64 +66,112 @@ class TimeSeriesExperiment(ExperimentalDesign):
     def __init__(
         self,
         data: pd.DataFrame,
+        outcome_variable_name: str,
+        variable_column: str,
```
Comment (ivyONS) [inline causalpy/pymc_experiments.py:70]: I know this came from the package, but any chance we can add documentation on the `data` format expecated? e.g. long format, index=time(?)

File: causalpy/pymc_experiments.py [hunk 2]
```diff
@@ -59,64 +66,112 @@ class TimeSeriesExperiment(ExperimentalDesign):
...
         treatment_time: Union[int, float, pd.Timestamp],
         formula: str,
         model=None,
         **kwargs,
     ) -> None:
         super().__init__(model=model, **kwargs)
         self._input_validation(data, treatment_time)
-
+        self.variable_column = variable_column
+        self.outcome_variable_name = outcome_variable_name
```
Comment (ivyONS) [inline causalpy/pymc_experiments.py:79]: Do we use the `outcome_variable_name`?

**Review by ivyONS — APPROVED** [review]
LGTM (reviewed changes, not the full code). 
Good design of the multi-variable representation and therefore nice flexibility with post analysis&plots.

File: causalpy/pymc_experiments.py (hunk 1 above)
Comment (SStock1) [reply causalpy/pymc_experiments.py:69]: I agree, I have added some documentation @ivyONS!

File: causalpy/pymc_experiments.py (hunk 2 above)
Comment (SStock1) [reply causalpy/pymc_experiments.py]: Yes, good spot, I should have taken this out now.

**Pull request merged**